### PR TITLE
Simplify and update the issue template

### PR DESCRIPTION
**Pull request by beorn7**
This got triggered by the introduction of GH discussions.

Signed-off-by: beorn7 <beorn@grafana.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,26 +1,8 @@
...
-	developer community first. This is in particular true for a request to
-	reconsider a prior rejection of a feature request or any of the declared
-	non-goals (see README.md). Those overarching discussions happen on the
-	developer mailing list (GitHub issues, in particular closed ones, are not
-	tracked by the wider developer community and thus inadequate):
-	https://groups.google.com/forum/#!forum/prometheus-developers
-	
-	You can find more information at: https://prometheus.io/community/
+	requests. For questions and discussions, please use GitHub discussions or
+	any of the other community channels: https://prometheus.io/community/
```
Comment (codesome) [inline .github/ISSUE_TEMPLATE.md:5]: Makes it sound like it is fine to open GitHub discussions for questions. I guess that is not what we want?

Comment (roidelapluie) [reply .github/ISSUE_TEMPLATE.md:5]: yes

**Review by roidelapluie — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (codesome) [reply .github/ISSUE_TEMPLATE.md:5]: Not sure what is that `yes` for since you also approved the PR.

Comment (codesome) [reply .github/ISSUE_TEMPLATE.md:5]: Oh I see Q&A in Discussions now :) I get your "yes" now

**Review by codesome — APPROVED** [review]

**Comment by beorn7** [comment]
Yeah, definitely Q&A's are fine in discussions.

**Pull request merged**